code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -329,11 +329,17 @@
 // ***************************************
 // *********** Queen Acidic Salve
 // ***************************************
-/datum/action/xeno_action/activable/psychic_cure/acidic_salve/queen
+/datum/action/xeno_action/activable/psychic_cure/queen_give_heal
```

Comment:
But then is there any reason this is a child of psychic cure as well

Reply:
There is, psychic cure has the `heal_range` var defined. Without it, I'd have to redefine it for `queen_give_heal` which feels unnecessary.